README.md

```diff
@@ -0,0 +1,34 @@
+## HOWTO build
```

Comment:
Please add a license header here as well (using an HTML comment to hide it).

Reply:
Done